docs/core/preview3/tools/csproj.md

```diff
@@ -25,81 +25,103 @@ see [the MSBuild project file](https://docs.microsoft.com/visualstudio/msbuild/m
 consumed but that they should not flow to the next project. 
 
 > [!NOTE]
 > This is a new term for project.json/xproj `SuppressParent` element. 
 
-The element can contain one or more of the following values:
+The attribute can contain one or more of the following values:
 
-* Compile – the contents of the lib folder are available to compile against.
-* Runtime – the contents of the runtime folder are distributed.
-* ContentFiles – the contents of the contentfiles folder are used.
-* Build – the props/targets in the build folder are used.
-* Native – the contents from native assets are copied to the output folder for runtime.
-* Analyzers – the analyzers are used.
+* `Compile` – the contents of the lib folder are available to compile against.
+* `Runtime` – the contents of the runtime folder are distributed.
+* `ContentFiles` – the contents of the contentfiles folder are used.
+* `Build` – the props/targets in the build folder are used.
+* `Native` – the contents from native assets are copied to the output folder for runtime.
+* `Analyzers` – the analyzers are used.
 
-Alternatively, the element can contain:
+Alternatively, the attribute can contain:
 
-* None – none of the assets are used.
-* All – all assets are used.
+* `None` – none of the assets are used.
+* `All` – all assets are used.
 
 ### DotnetCliToolReference
 `<DotnetCliToolReference>` element specifies the CLI tool that the user wants restores in the context of the project. It is 
 a replacement for the `tools` node in `project.json`. 
 
+```xml
+<DotnetCliToolReference Include="<package-id>" Version="" />
+```
+
 #### Version
-`<Version>` specifies the version of the package to restore. The element respect the rules of the NuGet versioning scheme.
+`<Version>` specifies the version of the package to restore. The attribute respect the rules of the NuGet versioning scheme.
 
 ### RuntimeIdentifiers
 The `<RuntimeIdentifiers>` element lets you specify a semicolon-delimited list of [Runtime Identifiers (RIDs)](../../rid-catalog.md) for the project. 
-RIDs enable publishing self-contained deployments. 
\ No newline at end of file
+RIDs enable publishing a self-contained deployments. 
+
+
+### RuntimeIdentifier
+The `<RuntieIdentifier>` elements allows you to specify only one [Runtime Identifier (RID)](../../rid-catalog.md) for the project. RIDs enable publishing a self-contained deployment. 
+
+### PackageTargetFallback 
+The `<PackageTargetFallback>` property allows you to specify a set of compatible targets to be used when restoring packages. They are designed to allow packages that use the dotnet TxM to operate with packages that don't declare a dotnet TxM. If your project is using the dotnet TxM then all the packages you depend on must also have a dotnet TxM, unless you add the `<PackageTargetFallback>` to your project in order to allow non dotnet platforms to be compatible with dotnet. 
```

Comment:
property --> element

"They are designed" --> "It is designed"

"is using the dotnet TxM" --> "uses the dotnet TxM,"